ConsoleApp/School/ClassRoom.cs

```diff
@@ -0,0 +1,8 @@
+namespace ConsoleApp.School;
+
+class ClassRoom
+{
+    public int RoomNumber { get; set; }
+    public int lesson{ get; set; }
```

Comment:
code convention violated here
```suggestion
    public int Lesson{ get; set; }
```